RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java

```diff
@@ -777,6 +783,37 @@ public void testAnd() {
     assertEquals(1, left.getLongCardinality());
     assertEquals(123, left.select(0));
   }
+  @Test
+  void testToArrayAfterAnd(){
+      Random random = new Random();
```

Comment:
I think this test case was not designed well to reproduce the case that the result `Roaring64Bitmap` has empty container after some `and` operations.Please redesign it.

Reply:
This test case just want to  test toArray function is correct after bitmap use and operation.I think empty container after and operations has no problem.Root cause is getLongIterator function was not considered this.

Reply:
I see what you mean. I'll modify the test case